### PR TITLE
Setting correct view on filter change #2668

### PR DESCRIPTION
**Pull request by petrican**

### CODE REVIEW

File: src/components/app/DocumentList/index.js [hunk 1]
```diff
@@ -424,7 +420,7 @@ class DocumentListContainer extends Component {
     filterView(windowType, viewId, filtersActive.toIndexedSeq().toArray())
       .then((response) => {
         const viewId = response.viewId;
-
+        this.connectWebSocket(viewId);
```
Comment (siemiatj) [inline src/components/app/DocumentList/index.js:423]: Do we have to do this for all filters or barcode scanner is just some edge case ?

**Comment by siemiatj** [comment]
Code-wise it's ok, but want to be sure about two things:
- are we unsubscribing the previous websocket when filtering ?
- do we have to re-subscribe for every filtering or just barcode scanner ?

File: src/components/app/DocumentList/index.js (hunk 1 above)
Comment (petrican) [reply src/components/app/DocumentList/index.js:423]: For all filters. If you look at the URI when you setup new filter the viewId is changing

**Comment by TheBestPessimist** [comment]
@siemiatj From my testing, it seems that each change in a filter will return a new view/viewID. 
I'm not sure if this needs the unsubscribe old -> subscribe new, but it is my understanding (from testing on local machine) that it should be like that, as since backend changed the view, the old one has stale data.

**Pull request merged**